### ISSUE TITLE
Compilation error in lammps-17Nov16 - Need to update force function

### ISSUE THREAD

**Issue by upayanbaul** [vasudevanv/pots#1]
Hi,

I am trying to compile with the WCA files. Compilation is exiting with error

../pair_lj_cut_coul_long_wca.cpp:679:44: error: too few arguments to function call, expected at least 6, have 4
  force->bounds(arg[0],atom->ntypes,ilo,ihi);
  ~~~~~~~~~~~~~                            ^
../force.h:128:3: note: 'bounds' declared here
  void bounds(const char *, int, char *, int, int &, int &, int nmin=1);
  ^

Could you please help?

Regards,
Upayan

**Comment by upayanbaul** [comment]
Hi,

The issue seems resolved on trying the following:
I checked that there is essentially no difference between the relevant parts in lammps versions 
from 2013 and the current version, except for the inclusions of the file to which the error message 
is printed. Thus I copied the "bounds" subroutine from a 2013 version of force.cpp (renamed it 
to boundsOLD) and included it in the current version. Accordingly, I call it from your code as

force->boundsOLD(arg[0],atom->ntypes,ilo,ihi);
instead of 
force->bounds(arg[0],atom->ntypes,ilo,ihi);

I understand that a better way probably would be to modify your subroutines, but my knowledge 
of c++ is non-existent. 

Sorry for the bother, but please comment if my way could lead to some error that I am failing 
to notice.

Regards,
Upayan

**Comment by upayanbaul** [comment]
Hi,

The issue is resolved. 
changing 
force->bounds(arg[0],atom->ntypes,ilo,ihi);
in your code to 
force->bounds(FLERR,arg[0],atom->ntypes,ilo,ihi);

Apologies for such a crap set of comments..

Regards,
Upayan

**Comment by vasudevanv** [comment]
> I understand that a better way probably would be to modify your subroutines

You are right. I last updated these in early 2014. I will update these to the latest version when I get some time closer to the new year's break.
